### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by mattfreeman-london**
updated README - please review and approve

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+- [Link to camera setup] (https://github.com/CamJam-EduKit/EduKit2/blob/master/CamJam%20Edukit%202%20-%20RPi.GPIO/CamJam%20EduKit%202%20-%20Sensors%20Worksheet%205%20(RPi.GPIO)%20-%20Movement.pdf)
+
+### APIs used
+- IFTTT with PushOver
+
+### Framework information
+- python version 2.7.9
+- rails version 5.1.5
+- Ruby gems - see Gemfile
+- python libraries: gpiozero, picamera, requests
```
Comment (SHUBV92) [inline README.md]: These lines need to start with the upper case, please let me know once done, so we can merge.

File: scrollphat/auth.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+consumer_key = 'WwXsQ5djrUXBE1I6QKCt3MvDQ'
+consumer_secret = '7Vlab5kTI4HtMdNUvMIdS1hyO1808OvRGVbqaehtbYF2KeQ1kw'
+access_token = '1213493117191512064-YYA2NamvFeBRofiSBdQ5i07rsY8hKB'
```
Comment (SHUBV92) [inline scrollphat/auth.py]: Delete this file

File: README.md [hunk 3]
```diff
@@ -1,3 +1,30 @@
-# RPI-Security-Sensor
-
+# Security Sensor
 ## Makers October 2019 Cohort Final Project
+
+A home security system, using a Raspberry Pi, motion sensor, and camera. Upon detecting motion, a picture is taken and sent to a hosted website. A notification is also sent to your phone, including a link to the picture URL.
+
+### Getting Started
```
Comment (SHUBV92) [inline README.md]: Start these lines with uppercase as well

File: README.md [hunk 4]
```diff
@@ -1,3 +1,30 @@
...
+- clone the project
+- build the hardware (see below for instructions)
+- set up the APIs
+- run the tests
+- visit the website & create an account
+
+### How to set up the motion sensor and camera
+- list of components:
```
Comment (SHUBV92) [inline README.md]: Start these lines with uppercase as well

File: README.md [hunk 5]
```diff
@@ -1,3 +1,30 @@
...
+  - Raspberry Pi (any model with wifi can be used)
+  - PIR Sensor on pin 4
+  - PiCamera
+  - female-to-female jumper wires
+- how to set up the motion sensor
```
Comment (SHUBV92) [inline README.md]: This one too

File: scrollphat/auth.py [hunk 6]
```diff
@@ -0,0 +1,4 @@
+consumer_key = 'WwXsQ5djrUXBE1I6QKCt3MvDQ'
```
Comment (SHUBV92) [inline scrollphat/auth.py]: Does this need to be on Github

File: README.md (hunk 3 above)
Comment (mattfreeman-london) [reply README.md]: none of these lines need to be changed - this is a paragraph, not bullet points

File: scrollphat/auth.py (hunk 6 above)
Comment (mattfreeman-london) [reply scrollphat/auth.py]: check again - have definitely deleted this on the branch now

**Pull request merged**